mmv1/templates/terraform/examples/sql_database_instance_postgres.tf.erb

```diff
@@ -0,0 +1,10 @@
+# [START cloud_sql_postgres_instance_80_db_n1_s2]
+resource "google_sql_database_instance" "<%= ctx[:primary_resource_id] %>" {
+  name             = "<%= ctx[:vars]['database_instance_name'] %>"
+  region           = "us-central1"
+  database_version = "POSTGRES_12"
+  settings {
+    tier = "db-custom-2-7680"
+  }
```

Comment:
I've added `deletion_protection = true` on the other `google_sql_database_instance` examples because

1. It is set to true by default, and without the ability to set it false in our tests, we'll keep spinning up instances and never delete them
2. I figured it was good to show users the functionality that their instances can't be deleted unless they opt-out of deletion protection

#1 is really just for my team, we could figure out a way around it if need be. Let me know if you think #2 is reasonable!